build/config/BUILDCONFIG.gn

```diff
@@ -279,6 +279,15 @@ if (target_os == "android") {
   assert(false, "Unsupported target_os: $target_os")
 }
 
+if (is_cobalt && _default_toolchain == host_toolchain) {
+  _toolchain_parts = string_split(_default_toolchain, ":")
```

Comment:
I just realized I was being dumb—instead of doing this string splitting at all, we should just prepend "cobalt/" to the string, so
```
_default_toolchain = "cobalt/" + _default_toolchain
```
which means we can put the toolchain definition in a different file without all this string manipulation. Sorry for all the back and forth on this!

Reply:
no worries :)